### PR TITLE
Consumable Treasure Stack Size

### PR DESCRIPTION
**Pull request by mhn65536**
Allows you to specify that treasurechests and npcs with consumables give you more than 1 item.

Recreated as branch PR.

### CODE REVIEW

File: FF1Lib/TreasureStacks.cs [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+using System.Threading.Tasks;
+
+namespace FF1Lib
+{
+	public enum TreasureStackSize
+	{
+		[Description("Default")]
+		Default,
+
+		[Description("Stingy (2 Tents, 5 Heals)")]
```
Comment (wildham0) [inline FF1Lib/TreasureStacks.cs:16]: Alright this looks good, just put the quantities in the tooltip and I'll merge this tomorrow.

File: FF1Blazorizer/Tabs/ShopsTab.razor [hunk 2]
```diff
@@ -16,6 +16,7 @@
 		<div class="checkbox-cell"></div>
 		<h4>Consumables</h4>
 		<EnumDropDown UpdateToolTip="@UpdateToolTipID" Id="startingItemsDropDown" TItem="StartingItemSet" @bind-Value="Flags.StartingItemSet">Starting Inventory:</EnumDropDown>
+		<EnumDropDown UpdateToolTip="@UpdateToolTipID" Indent TItem="TreasureStackSize" @bind-Value="Flags.ConsumableTreasureStackSize">TreasureChest Stacks: </EnumDropDown>
```
Comment (wildham0) [inline FF1Blazorizer/Tabs/ShopsTab.razor:19]: No need for the Indent

File: FF1Lib/TreasureStacks.cs (hunk 1 above)
Comment (mhn65536) [reply FF1Lib/TreasureStacks.cs]: ok

File: FF1Blazorizer/Tabs/ShopsTab.razor (hunk 2 above)
Comment (mhn65536) [reply FF1Blazorizer/Tabs/ShopsTab.razor]: ok

**Pull request merged**